app/src/main/java/org/fossasia/openevent/general/auth/ProfileFragment.kt

```diff
@@ -103,6 +108,7 @@ class ProfileFragment : Fragment(), BottomIconDoubleClick {
             .nonNull()
             .observe(viewLifecycleOwner, Observer {
                 rootView.snackbar(it)
+                profileViewModel.syncProfile()
```

Comment:
Why this line?

Reply:
To update verification status in profile section by fetching profile from the server.

Reply:
Okay, it's duplicate. Removing.

Reply:
Updated